pweave/scripts.py

```diff
@@ -50,6 +62,12 @@ def weave():
     if options.figformat is not None:
         opts_dict["figformat"] = ('.%s' % options.figformat)
 
+    if opts_dict.pop('verbosity'):
+        logging.getLogger("ipykernel").propagate = True
```

Comment:
line should be split into two (with ipykernel_logger intermediate variable)

Reply:
Same here, please don't be terse with your comments, I am not as quick. You could use insert suggestion feature (little + in a card stack above)

Reply:
```suggestion
        logging.getLogger("ipykernel").propagate = True
```